draft-ietf-quic-multipath.md

```diff
@@ -355,7 +356,7 @@ This proposal adds four multipath control frames for path management:
 (see {{path-abandon-frame}}),
 - PATH_BACKUP and PATH_AVAILABLE frames to express a preference
 in path usage (see {{path-backup-frame}} and {{path-available-frame}}), and
-- MAX_PATH_ID frame for increasing the limit of active paths.
+- MAX_PATH_ID frame for increasing the limit of open paths.
```

Comment:
Just here that I think the "open" word is a bit misleading (but original text was probably)
```suggestion
- MAX_PATH_ID frame for increasing the limit of usable path identifiers.
```